From 52c78df67a08de074991a93b57946b7bd5ea7196 Mon Sep 17 00:00:00 2001
From: Dima Pasechnik <dimpase@gmail.com>
Date: Fri, 8 May 2020 15:53:25 +0100
Subject: [PATCH 1/1] remove redundant 1st and last lines

they remain if the script is run under ksh, leading to broken .pc file
---
 linbox.pc.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/linbox.pc.in b/linbox.pc.in
index f54285e..eb6835b 100644
--- a/linbox.pc.in
+++ b/linbox.pc.in
@@ -1,4 +1,3 @@
-/------------------ linbox.pc ------------------------
 prefix=@prefix@
 exec_prefix=@prefix@
 libdir=@libdir@
@@ -11,4 +10,4 @@ Version: @VERSION@
 Requires: fflas-ffpack >= 2.4.0, givaro >= 4.1.0
 Libs: -L${libdir} -llinbox @LINBOXSAGE_LIBS@ @NTL_LIBS@ @MPFR_LIBS@ @FPLLL_LIBS@ @IML_LIBS@ @FLINT_LIBS@ @OCL_LIBS@
 Cflags: @DEFAULT_CFLAGS@ -DDISABLE_COMMENTATOR -I${includedir} @NTL_CFLAGS@ @MPFR_CFLAGS@ @FPLLL_CFLAGS@  @IML_CFLAGS@ @FLINT_CFLAGS@ 
-\-------------------------------------------------------
+
-- 
2.26.2

